documentation/index.rst

```diff
@@ -15,6 +15,17 @@ available in the command line interface.
    content/tests
    content/codingstyle
 
+kw Tutorials
+============
+
+Check out our tutorials to see how you can use kw.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Tutorials:
+
+   tutorials/buildlinux
+
```

Comment:
![section](https://user-images.githubusercontent.com/3153524/120563109-cfff5680-c3d5-11eb-81ec-44e297411dba.png)

Idk what happened here, but it looks like that `kw man` section disappeared. Can we make sure that we have a clear separation for kw man? Feel free to propose anything different.

Reply:
@rodrigosiqueira I added a new commit for creating a section for the manual.